specs/encode_repeating_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require_relative '../lib/encode_repeating'
+
+describe "encode repeating" do
+  describe "basic tests" do
+    describe "test 1" do
```

Comment:
Same as the other repos, we should probably remove the `describe` blocks that are immediate parents of individual tests.